### PR TITLE
Add workload for Data Engineering workshop

### PR DESCRIPTION
**Pull request by rimolive**
##### SUMMARY
Include a workload for the Data Engineering workshop

##### ISSUE TYPE
- New config Pull Request

##### COMPONENT NAME
Open Data Hub operator

##### ADDITIONAL INFORMATION
N/A

### CODE REVIEW

File: ansible/roles/ocp4-workload-data_engineering_workshop/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+# Default variables that are defined by the playbook that will be deploying these roles
+become_override: False
+ocp_username: opentlc-mgr
+silent: False
+
+# This workshop requires per user 10-12 CPU and 16GB of Memory
+# For a 50 person workshop, you'll need ~40 OCP worker nodes running on with m4.4xlarge instances
+_infra_node_replicas: 40
```
[the remark this answers is not in the document]
Comment (rimolive) [reply ansible/roles/ocp4-workload-data_engineering_workshop/defaults/main.yml]: All I did was copy and paste one existing workload to create my own. This came from other workload script: https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles/ocp4-workload-rhte-analytics_data_ocp_workshop/defaults/main.yml#L9

If that's inaccurate, so all the other workloads might have the same configuration

File: ansible/roles/ocp4-workload-data_engineering_workshop/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+_infra_node_instance_type: m4.4xlarge
```
[the remark this answers is not in the document]
Comment (rimolive) [reply ansible/roles/ocp4-workload-data_engineering_workshop/defaults/main.yml]: This is a for a workshop which if we can get as many resources possible, the better

**Comment by stencell** [comment]
I'm closing this since it has been untouched for 3 months.

**Comment by rimolive** [comment]
Why are you closing this issue? I requested more info about if the changes are really necessary and got no answer so far.

**Comment by stencell** [comment]
@rimolive You are welcome to reopen it. I don't really see any questions, and again it has been untouched for 3 months. I think @wkulhanek made the comment that the defaults you were choosing (40x nodes) were quite a lot. Maybe you can come up with a smaller default and if you do need it to be bigger, that becomes variable based on the catalog item and config?

**Pull request closed without merging**